### ISSUE TITLE
It export html and other useless information

### ISSUE THREAD

**Issue by jiansongjay** [kartik-v/yii2-export#102]
The export-menu export some html info which I don't need。Like something below。
![image](https://cloud.githubusercontent.com/assets/2103028/12077181/595dd654-b211-11e5-8b4d-72bb4e05a15b.png)

**Comment by kartik-v** [comment]
You need to check your markup...

**Comment by maxxer** [comment]
@jiansongjay did you manage to fix your issue? I'm having the same and I've been pulling my hair for hours, unfortunately without success.

I've checked HTML markup but except from minor invalid params it doesn't seem to contain unreferenced tags. I tried stripping down view and layout to the bare minimum essential and the best I could get is still a CSV with [BOM char feff](https://forum.yiiframework.com/t/yii2-export-exported-csv-xls-contains-html-from-page/124264/5). I don't really know what else to look for.

A small excerpt of the HTML is [this one](https://paste.ufficyo.com/?07c58496a06fae07#3UJ4+znTiFrbXqorFGOpk0FgN7ft4n5EcNvY/JdvpKU=).

If you have any hint it's welcome! thanks
